qa/tasks/rgw.py

```diff
@@ -260,18 +245,24 @@ def start_rgw(ctx, config, on_client = None, except_client = None):
     """
     Start rgw on remote sites.
     """
-    log.info('Starting rgw...')
+    log.info('ALI ADDED Starting rgw...')
```

Comment:
Yes i plan to do a cleanup of the entire PR. I just add that in so I can easily ctrl+f important parts of the logs.